README.md

```diff
@@ -119,17 +136,17 @@ Please see information about our [work flow](./docs/BRANCHING.md) before proceed
 5. Create the pull request.
 
 ## Contributors
----
+
 Thanks to the following people who have contributed to this project:
 
 * [@tjayrush](https://github.com/tjayrush)
 * [@crodnun](https://github.com/crodnun)
 * [@wildmolasses](https://github.com/wildmolasses)
 
 ## Contact
----
+
 If you have specific requests, contact us here <info@quickblocks.io>.
 
 ## License
----
-This project licensed under the [Apache License Version 2.0](LICENSE.md).
+
+This project licensed under the (GNU General Public License v3.0)(LICENSE.md).
```

Comment:
This is actually not correct. There are actually two licenses here. Most of the code (./libs, ./dev_tools, ./tools, ./examples) should be licensed under Apache (or GNU if that's not possible). All the code under ./apps is not licensed for open source.  I might change this to 'Parts of this repo are licensed under....'